docs/oem-provisioning-24_04_1.md

```diff
@@ -117,7 +117,44 @@ autoinstall:
 EOF
 touch meta-data
 ```
+#### Adding additional Language packs
 
+The default Ubuntu ISO only has the English language pack seeded. To have additional languages
+available in Gnome Initial Setup, you will need to install additional language packs via the
+`autoinstall.yaml`
+
+To install additional language packs, you can make use of [late commands](https://canonical-subiquity.readthedocs-hosted.com/en/latest/reference/autoinstall-reference.html#late-commands) in the `autoinstall.yaml:
+
+```yaml
+#cloud-config
+autoinstall:
+  version: 1
+  late-commands:
+    - |
+      curtin in-target --target=/target -- bash -c '
+        # Update package list
+        apt-get update
+
+        # Installing languages not in seed
+        # Add language codes of packs you with to install here 
+        for lang in "fr" "de"; do
+            for pkg in $(check-language-support --show-installed -l "$lang"); do
+                # gimp: not installed by default
+                # fcitx5: not used by ubuntu as gnome is in use
+                # mozc-utils-gui: is in universe
+                if [ "${pkg%%-*}" = gimp ] || [ "${pkg%%-*}" = fcitx5 ] || [ "$pkg" = "mozc-utils-gui" ]; then
+                    continue
+                fi
+                apt-get install -y "$pkg"
+            done
+            for pkg in $(check-language-support --show-installed -l "$lang" -p "ibus,gnome-user-docs,gvfs"); do
+                apt-get install -y "$pkg"
+            done
+            for pkg in $(check-language-support --show-installed -l "$lang" -p "libreoffice-common"); do
```

Comment:
We don't install libreoffice by default (unless you change the option during the installation to install extra software) so that should probably be conditional on whether libreoffice-common is installed?

Reply:
Ah, and additionally, don't we install the snap by default if you choose the normal install? Should I actually always skipping libre office related language support?